pkg/networkmanager/firewall.jsx

```diff
@@ -860,14 +861,14 @@ class ActivateZoneModal extends React.Component {
                             <Checkbox key={i.device}
                                       id={i.device}
                                       value={i.device}
-                                      onChange={(value, event) => this.onInterfaceChange(event)}
+                                      onChange={(event, value) => this.onInterfaceChange(event)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-112552-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)